src/lib.rs

```diff
@@ -488,6 +488,16 @@ impl FixedBitSet {
         }
     }
 
+    /// In-place union of two `FixedBitSet`s.
+    ///
+    /// This method will not grow the capacity of `self` to match `other`'s.
+    /// If `other`'s capacity is larger than `self`'s, the extra bits will be ignored.
+    pub fn union_with_unchecked(&mut self, other: &FixedBitSet) {
```

Comment:
`_unchecked` is usually reserved for `unsafe` functions, and eschewing convention here is probably not the best UX. That said, I'm struggling to come up with a better name.

Reply:
I had the same thought but couldn't come up with a better name at the time either.

Some candidates I'm thinking of now:
- `union_with_no_resize`
- `union_with_skip_resize`
- `union_with_direct`
- `union_with_exact`

Let me know if you think of a different one too and I'm happy to change it